packages/devextreme/js/__internal/grids/grid_core/sticky_columns/const.ts

```diff
@@ -0,0 +1,12 @@
+export enum StickyPosition {
+  Left = 'left',
+  Right = 'right',
+}
```

Comment:
Minor, current best-practice in our codebase is to use 'union type' instead of 'enum':

- `export enum StickyPosition {...}` > `export type StickyPosition = 'left' | 'right'`
- `StickyPosition.right` -> `'right'`